certified-connectors/Kanbanize/Readme.md

```diff
@@ -0,0 +1,21 @@
+
+## Kanbanize
+An official connector working with the Kanbanize API.
```

Comment:
When the description of the connector is updated, it'd be great to include some of those changes to what is discussed here. Again, it's helpful to cover the main purpose and value offered by the connector.